### DISCUSSION TITLE
New ReFS feature in Windows 11 24H2

### DISCUSSION THREAD

**Discussion by HotCakeX** [HotCakeX/Harden-Windows-Security#260] [Announcements]
The `Supports Encrypted File System` is now available in 24H2 build for [ReFS](https://learn.microsoft.com/en-us/windows-server/storage/refs/refs-overview) formatted drives. That could be referring to the [EFS](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc721923(v=ws.10)?redirectedfrom=MSDN) or something else.

* Build 23H2 ReFS version: 3.10
* Build 24H2 ReFS version: 3.14

You can verify it using the following commands and get more info about your ReFS formatted volumes. Replace `D:` with your own drive letter.

```
fsutil fsinfo volumeinfo D:
fsutil fsinfo refsinfo D:
```

**Comment by 13xforever** [comment]
They call it [Personal Data Encryption ](https://www.microsoft.com/en-us/security/blog/2024/05/20/new-windows-11-features-strengthen-security-to-address-evolving-cyberthreat-landscape/#stay-ahead-of-evolving-threats-with-windows). And [documentation](https://learn.microsoft.com/en-us/windows/security/operating-system-security/data-protection/personal-data-encryption/) states it was first implemented in 22H2 (but maybe not for ReFS specifically).

**Reply by HotCakeX** [reply]
Thank you! Yes, on my 23H2 build it doesn't exist so i think it's a 24H2 feature. The official ReFS document is very outdated, I made a [PR](https://github.com/MicrosoftDocs/windowsserverdocs/pull/7868) for it.

**Comment by Karl-WE** [comment]
Hi @HotCakeX the ReFS gist has moved where you commented.
Link in the reply above yours.

The compression feature is documented on old and new gist.

Windows Server 2025 will have a new dedup including ReFS compression for even higher effiency, e. g. on Azure Stack HCI 23H2, S2D, etc.

**Reply by HotCakeX** [reply]
Nice, now all we want is to make it bootable :)

**Reply by Karl-WE** [reply]
Bootable ReFS is there but the use case is yet limited to a certain thing. No Client / Server OS yet. Unfortunately.

**Reply by HotCakeX** [reply]
Oh! if client/server OS doesn't support it then what does? 🤔like HoloLens, IoT stuff?
